browser/resources/settings/brave_tor_page/brave_tor_subpage.html

```diff
@@ -0,0 +1,76 @@
+<style include="settings-shared md-select">
```

Comment:
It looks like the Tor settings are being moved to a new Tor subpage directly under the IPFS subpage. Would it make more sense perhaps to move this to the *Privacy & Security* sub-page (e.g. just above the *Safety check* section in
`brave://settings/privacy`)? It would avoid putting too much in the top-level list of settings and it's a natural fit for *Privacy & Security*.

Reply:
It's already done.
![image](https://user-images.githubusercontent.com/73575789/180365738-82baf911-53a4-4a99-8b7d-9719a9d504ad.png)